maven-plugin/src/main/java/org/openmrs/maven/plugins/RemoveDependency.java

```diff
@@ -0,0 +1,61 @@
+package org.openmrs.maven.plugins;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.openmrs.maven.plugins.model.DistroProperties;
+
+import java.io.File;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+@Mojo(name = "rm-dependency", requiresProject = false)
```

Comment:
I'd prefer this was just named `remove`. I don't think `rm-dependency` is very intuitive of a name...